pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[project]
+name = "pulumi-datarobot-utils"
+dynamic = ["version"]
+description = "A set of Pulumi CustomResources and other things on top of the pulumi-datarobot provider."
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.11"
+maintainers = [
+    { name = "Lukas Innig", email = "lukas.innig@datarobot.com" },
```

Comment:
I'm curious if it's worth also adding a generic email somewhere like:
`api-maintainer@datarobot.com`

https://github.com/datarobot/public_api_client/blob/070241a9a21b5bf19ccaaa3163b59741a8c5f3d6/common_setup.py#L184C19-L186